src/renderRequestElement.ts

```diff
@@ -0,0 +1,29 @@
+import {QROptions, RequestData, ShouldRenderButton, RequestElementResult} from './types'
+import {renderRequestButton} from './elements/renderRequestButton'
+import {renderRequestQRCode} from './elements/renderRequestQRCode'
+
+const bowser = require('bowser')
+
+const renderRequestElement = (config: {
+  container: HTMLElement
+  requestData: RequestData
+  qrOptions?: Partial<QROptions>
+  shouldRenderButton?: ShouldRenderButton
+}): RequestElementResult => {
+  if (config.shouldRenderButton === undefined) {
+    config.shouldRenderButton = parsedResult => {
+      const isMobile = parsedResult.platform.type === 'mobile' || parsedResult.platform.type === 'tablet'
+      const isIOS = () => parsedResult.os.name === 'iOS'
```

Comment:
Is there a reason why this isn't a boolean? I'd pick either boolean or function to be consistent if not.